src/matrix.rs

```diff
@@ -303,8 +303,9 @@ impl Matrix4 {
         )
     }
 
-    /// Create a 2d skew matrix
-    pub fn create_skew(sx: f32, sy: f32) -> Matrix4 {
+    /// https://drafts.csswg.org/css-transforms/#funcdef-skew
+    pub fn create_skew(alpha: f32, beta: f32) -> Matrix4 {
+        let (sx, sy) = (beta.tan(), alpha.tan());
```

Comment:
I see this was addressed [previously](https://github.com/servo/euclid/pull/77#discussion_r31617589).